eclass/vala.eclass

```diff
@@ -175,4 +182,11 @@ vala_src_prepare() {
 	export PKG_CONFIG_PATH="${T}/pkgconfig:${PKG_CONFIG_PATH}"
 }
 
+# @FUNCTION: vala_src_prepare
+# @DESCRIPTION:
+# Exported phase function in EAPIs 6 and 7. Calls vala_setup.
+if [[ ${EAPI} == [67] ]]; then
```

Comment:
Thank you for doing this. I will finally be able to remove inappropriate comments from my ebuilds!